src/Microsoft.Azure.WebJobs.ServiceBus/Listeners/ServiceBusQueueListenerFactory.cs

```diff
@@ -1,46 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
didn't find that these listener factories provided much value besides testability, so canned them.